bower.json

```diff
@@ -20,7 +20,7 @@
   ],
   "dependencies": {
     "agrc": "agrc-widgets#~6.1.4",
-    "ijit": "agrc-ijit#~3.2.2",
+    "ijit": "agrc-ijit#~3.2.3-rc.1",
```

Comment:
this could be set to @master or bb40f8e894f8f45d4c08222b84ffd0c1074f5891 if you wanted to get the latest bits